core/shared/src/test/scala/fs2/StreamSpec.scala

```diff
@@ -2125,6 +2125,7 @@ class StreamSpec extends Fs2Spec {
       }
 
       "resources acquired in outer stream are released after inner streams complete" in {
+        pending
```

Comment:
Odd, these are passing on series/1.x AFAIK

Reply:
I'm commenting them out for now, but may re-add them after I address the fact that flickersOnTravis is not behaving as expected (does not exclude test on Travis like it should)